server/sandbox_run_linux.go

```diff
@@ -698,6 +692,42 @@ func (s *Server) runPodSandbox(ctx context.Context, req *pb.RunPodSandboxRequest
 		return nil, err
 	}
 
+	// now that we have the namespaces, we should create the network if we're managing namespace Lifecycle
+	var ips []string
+	var result cnitypes.Result
+
+	if s.config.ManageNSLifecycle {
+		ips, result, err = s.networkStart(ctx, sb)
```

Comment:
If it is time consuming, would it make sense to do it in a parallel go routine and join it at the end of the function?

Generally speaking, does a sandbox have to be ready immediately after the request happened? I would assume that the kubelet checks via the PLEG if the sandbox is ready in any case. This way we could do everything in the background (in theory).